### PR TITLE
[feature][ui]Alert plugin design

### PR DESCRIPTION
**Pull request by break60**
issue #3049

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/router/index.js [hunk 1]
```diff
@@ -397,6 +397,14 @@ const router = new Router({
           meta: {
             title: `${i18n.$t('Token manage')}`
           }
+        },
+        {
+          path: '/security/Alarm-plugin-example',
+          name: 'Alarm-plugin-example',
+          component: resolve => require(['../pages/security/pages/alarmPluginExample/index'], resolve),
```
Comment (gaojun2048) [inline dolphinscheduler-ui/src/js/conf/home/router/index.js:404]: can you change `alarm` to `alert` ?

**Review by gaojun2048 — CHANGES_REQUESTED** [review]
1、GroupType should be delete.
2、And can you change `alarm` to `alert` ?
3、Can you show the page image generated by the front-end code? These  can help us review these codes more conveniently. Thank you very much.（可以展示一下这个pr生成的前端页面的图片吗？这将帮忙我们更方便的review前端的代码，非常感谢）。

**Comment by break60** [comment]
At present, only a new instance page of the alarm plug-in is added. The fields in the list are copied from other pages. The back end can be coordinated and adjusted in the return field.

File: dolphinscheduler-ui/src/js/conf/home/router/index.js (hunk 1 above)
Comment (break60) [reply dolphinscheduler-ui/src/js/conf/home/router/index.js:404]: Okay, because the local library is re-forked, merge first, and then I modify it together

Comment (gaojun2048) [reply dolphinscheduler-ui/src/js/conf/home/router/index.js:404]: ok

**Review by gaojun2048 — COMMENTED** [review]
+1

**Review by gaojun2048 — APPROVED** [review]

**Review by qiaozhanwei — APPROVED** [review]
+1

**Pull request merged**